### PR TITLE
Add documentation page for google tag module

### PR DESCRIPTION
**Pull request by r0b1n**
Hey team,

The module described on the new page is being reviewed by the Marketplace team and not yet published (the link doesn't work), but the link itself must be correct. It is expected to be live today or tomorrow. This documentation page could be merged as soon as it is good to go.

Thank you!

### CODE REVIEW

File: content/en/docs/appstore/modules/google-tag.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+{{% alert color="warning" %}}
+When using the Google Tag widget with Mendix Studio Pro 8, there is a limitation where non-relevant properties are not hidden in the configuration window (for example, advanced properties are not hidden when selecting **Basic** mode). This issue does not affect the functionality of the widget, but it makes harder to configure. See the sections below to determine which properties need to be configured depending on the mode of the widget.
+{{% /alert %}}
+
+## 2 Google Tag Widget
+
+### 2.1 Basic Mode {#widget-basic-mode}
+
+"When the **Widget mode** is set to **Basic**, the widget only sends page tracking events, which covers the majority of use cases. In this mode, only the **Tag ID** property needs to be filled in. Additionally, you can fill in the **Parameters** if you want to include specific information along with the page view events. For information on how to configure Parameters, please refer to the [Parameters](#widget-advanced-parameters) section below."
```
Comment (Adam-Dupaski) [inline content/en/docs/appstore/modules/google-tag.md]: @r0b1n is there a reason there are quote marks `"` `"` used around this paragraph?

**Comment by Adam-Dupaski** [comment]
@r0b1n thanks for the PR, please see above comment, and please let me know when it's released in order to publish.

File: content/en/docs/appstore/modules/google-tag.md (hunk 1 above)
Comment (r0b1n) [reply content/en/docs/appstore/modules/google-tag.md]: Nope, they are there by mistake.

**Comment by r0b1n** [comment]
@Adam-Dupaski thank you! The module is approved and [released](https://marketplace.mendix.com/link/component/207519). This documentation page should go live as well.

**Review by Adam-Dupaski — APPROVED** [review]

**Pull request merged**